candidate_models/model_commitments/vonecornets.py

```diff
@@ -0,0 +1,93 @@
+        region = list(regions)[0]
+
+        if '.' in region:
+            region = region.split('.')[1]
+        if region == 'vone_block':
+            region = 'V1'
+
+        time_bins = [self.time_mapping[region][timestep] if timestep in self.time_mapping[region] else (None, None)
+                     for timestep in responses['time_step'].values]
+        responses['time_bin_start'] = 'time_step', [time_bin[0] for time_bin in time_bins]
+        responses['time_bin_end'] = 'time_step', [time_bin[1] for time_bin in time_bins]
+        responses = NeuroidAssembly(responses.rename({'time_step': 'time_bin'}))
+        responses = responses[{'time_bin': [not np.isnan(time_start) for time_start in responses['time_bin_start']]}]
+        # select time
+        time_responses = []
+        for time_bin in tqdm(self.recording_time_bins, desc='CORnet-time to recording time'):
+            time_bin = time_bin if not isinstance(time_bin, np.ndarray) else time_bin.tolist()
+            time_bin_start, time_bin_end = time_bin
+            nearest_start = find_nearest(responses['time_bin_start'].values, time_bin_start)
+            bin_responses = responses.sel(time_bin_start=nearest_start)
+            bin_responses = NeuroidAssembly(bin_responses.values, coords={
+                **{coord: (dims, values) for coord, dims, values in walk_coords(bin_responses)
+                   if coord not in ['time_bin_level_0', 'time_bin_end']},
+                **{'time_bin_start': ('time_bin', [time_bin_start]),
+                   'time_bin_end': ('time_bin', [time_bin_end])}
+            }, dims=bin_responses.dims)
+            time_responses.append(bin_responses)
+        responses = merge_data_arrays(time_responses)
+        return responses
+
+
+def vonecornet_s_brainmodel():
+    # map region -> (time_start, time_step_size, timesteps)
+    time_mappings = {
+        'V1': (50, 100, 1),
+        'V2': (70, 100, 2),
+        # 'V2': (20, 50, 2),  # MS: This follows from the movshon anesthesized-monkey recordings, so might not hold up
+        'V4': (90, 50, 4),
+        'IT': (100, 100, 2),
+    }
+    return VOneCORnetCommitment(identifier='CORnet-S', activations_model=vonecornet('cornets'),
+                                layers=['vone_block.output-t0'] + [f'model.{area}.output-t{timestep}'
+                                                                   for area, timesteps in
+                                                                   [('V2', range(2)), ('V4', range(4)), ('IT', range(2))]
+                                                                   for timestep in timesteps] +
+                                       ['model.decoder.avgpool-t0'], time_mapping=_build_time_mappings(time_mappings))
+
+
+class VOneCORnetBrainPool(UniqueKeyDict):
+    def __init__(self):
+        super(VOneCORnetBrainPool, self).__init__(reload=True)
+
+        model_pool = {
+            'vonecornets': LazyLoad(vonecornet_s_brainmodel),
```

Comment:
would do `vonecornet-s` or `VOneCORnet-S`

Reply:
Agree. Let me know if you agree with the previous 2 suggestions and I can submit and updated version.

Reply:
to make it consistent with the [CORnet naming](https://github.com/brain-score/candidate_models/blob/73a62b7d9416be13cde864531bba122ab991fe2b/candidate_models/model_commitments/cornets.py#L347), let's use `VOneCORnet-S`?